src/oc_erchef/priv/reindex-opc-organization

```diff
@@ -41,9 +41,19 @@
 %%   2 - Invalid organization name given
 %%
 main(Args) ->
-    init_network(),
-    [Command, OrgInfo, Context] = validate_args(Args),
-    perform(Command, Context, OrgInfo).
+    try
+      init_network(),
+      [Command, OrgInfo, Context] = validate_args(Args),
+      perform(Command, Context, OrgInfo)
+    catch 
+        error:{badmatch,{error,{conn_failed,{error,econnrefused}}}} ->
+            io:format("~n Unable to start new erlang node due to network issue~n Please check if the localhost is resolved to 127.0.0.1 or check if firewall is blocking the connection~n");
+        error:{badmatch,{error,{already_started,_PID}}} ->
+            io:format("~n The reindexing task is running in other process~n Please kill the other process and restart the Chef Infra server~n");
```

Comment:
Could you add the command to find and kill the other process?

Reply:
Updated the same